scripts/SREP/infra-node-resize/script.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+# This script checks if any tech preview features are enabled through feature sets.
+
+set -e
+set -o nounset
+set -o pipefail
+
+# fetch the infra machinepools
+get_machinepool() {
+    echo 'FETCHING INFRA MACHINEPOOLS...'
+    NAMESPACE=$(oc get ns | grep "$CLUSTER_ID" | awk '{print $1}')
+    oc -n "$NAMESPACE" get machinepool
+    INFRA_MACHINEPOOL=$(oc -n "$NAMESPACE"  get machinepool -o json | jq -r '.items[] | select(.spec.name=="infra") | .metadata.name')
+    oc -n "$NAMESPACE" get machinepool "$INFRA_MACHINEPOOL" -o json | jq -r ".spec.platform.type"
+}
+
+# update the yaml file
+update_cluster_infra_yaml() {
+    echo 'UPDATING THE YAML FILE FOR TEMPORARY MACHINEPOOL...'
+    oc -n "$NAMESPACE" get machinepool "$INFRA_MACHINEPOOL" -o yaml > /tmp/cluster-infra.yaml
+    yq -i 'del(.metadata.creationTimestamp) | del(.metadata.finalizer) | del(.metadata.resourceVersion) | del(.metadata.generation) | del(.metadata.selfLink) | del(.metadata.uid) | del(.status)' /tmp/cluster-infra.yaml
+    cat /tmp/cluster-infra.yaml 
```

Comment:
why need this step?

Reply:
To show if it has been correctly updated